cmd_gen_finetuning.ipynb

```diff
@@ -121,22 +123,28 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "from transformers import BitsAndBytesConfig\n",
     "from unsloth import FastLanguageModel\n",
     "\n",
+    "max_seq_length = 2048\n",
+    "random_seed = 42\n",
+    "\n",
+    "\n",
     "# configure quantization method for base model\n",
     "quantization_config = BitsAndBytesConfig(\n",
-    "    load_in_8bit=True,\n",
+    "    load_in_4bit = True,\n",
```

Comment:
Why change from 8bit to 4bit? A100s support 8bit and Llama 3.1 8B fits comfortably into its memory with this repesentation.

Reply:
because the config we used had bad performance as opposed to the Unsloth tutorial on our test bot. Hence we changed the config to the Unsloth's to have a better first-try config for our customers